linux/user-and-file-management/user-permissions/file-ownership.md

```diff
@@ -42,33 +28,26 @@ On Linux, every file and directory is given both a user owner and a group owner.
 Here is some example output:
 
 ```bash
-$ ls -ld /var/log
```

Comment:
The point of `$` was to convey the fact that the command should be ran in a terminal. why remove it?

Reply:
users found it confusing and thought that the `$` should be included in the command.